consensus/core/src/dag_state.rs

```diff
@@ -764,9 +784,19 @@ impl DagState {
             if commit.timestamp_ms() < last_commit.timestamp_ms() {
                 panic!("Commit timestamps do not monotonically increment, prev commit {:?}, new commit {:?}", last_commit, commit);
             }
+            commit
+                .timestamp_ms()
+                .saturating_sub(last_commit.timestamp_ms())
         } else {
             assert_eq!(commit.index(), 1);
-        }
+            commit.timestamp_ms()
```

Comment:
Which this instead of `0`?

Reply:
Yeah you are right - don't know why I've used the commit timestamp here.